### ISSUE TITLE
Tailor Signer Node rewards towards greater dApp and community adoption

### ISSUE THREAD

**Issue by ghost** [gochain/gip#25]
The driver behind this idea is to encourage adoption of the Gochain network.

**Assumptions:**
Total Go supply                                                            = 1,000,000,000 (1 Billion)
Signer Node Rewards                                                  = 4.5% of total supply adjusted per year.
Estimated amount of Go towards nodes per year:      = 45000000 (45 Million)

Assuming 50 nodes reached after 12 months: 900,000 GO per node per year.

At a current conservative valuation of 0.05c per GO, the total dollar value that signer nodes are rewarded with = $2.25 million per year.

This suggestion is to break up the 4.5% into a few different components that may encourage network / protocol adoption as follows:

**Once a company / enterprise is approved as a signer node, they receive rewards on the following basis:**

Only 90% of the 45 million GO to be used for signer nodes: (40,500,000 per year at the current rate)

- Year 1 - 50% of rewards     -     50% of reward towards dApp development fund.

- Year 2 - 60% of rewards     -     40% of reward towards dApp development fund.

- Year 3 - 70% of rewards     -     30% of reward towards dApp development fund.

- Year 4 - 80% of rewards     -     20% of reward towards dApp development fund.

- Year 5 - 100% of rewards     -   00% of reward towards dApp development fund.

The remaining 10% of the signer node rewards can be used in a number of ways in order to encourage further community adoption / or even some form of CSR.

- Airdrops to existing community members based on current Go allocation.

- Gochain sponsored community events. University Scholarships (Blockchain based). 

- Other ideas welcome.

**To be discussed:**

- How the Gochain dApp fund can be managed in a transparent manner. 
- Should VC's be involved / entrusted with the funds.
- Should VC's  be engaged with only on the basis that they also contribute 50% towards the dApp start-up that is to develop on Gochain.

**In Summary:**
The 5 year reward system should encourage signer nodes to act in the interest of the Gochain network at all times and be in the for the long run as rewards increase with time up and until the 5 year limit when 100% of rewards are paid out.

As the Gochain network grows, it is also important to understand how the existing nodes run by Gochain use the rewards accumulated by their own nodes. The community has no doubt that these funds are used towards building out partnerships / enhance the eco-system, but in the order to avoid unnecessary negative scrutiny in the future it would be helpful to have a single rule that fits all signer nodes in this regard.

Open to further discussion / suggestions.
